Lib/test/test_importlib/builtin/test_finder.py

```diff
@@ -1,13 +1,13 @@
 from test.test_importlib import abc, util
 
-machinery = util.import_importlib('importlib.machinery')
+machinery = util.import_importlib("importlib.machinery")
```

Comment:
It there the point to change single quotes to the double ones across the whole file, or was it done by an IDE autofomatter?

Reply:
Totally accident that I thought I had reverted before pushing the PR.